server/routerlicious/packages/services/src/throttler.ts

```diff
@@ -60,6 +61,14 @@ export class Throttler implements IThrottler {
                     eventName: "throttling",
                 },
             });
+            Lumberjack.info(`Throttled: ${id}`, {
```

Comment:
Let's remove the messageMetadata "wrapper", and instead use the common property keys like
```[CommonProperties.telemetryGroupName]: "throtting",
[ThrottlingTelemetryProperties.key]: throttleId
```